src/reformat_libyuv.c

```diff
@@ -461,6 +501,275 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
+
+        if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV444) {
+            // This doesn't currently exist in libyuv
+#if 0
+            if (I410ToRGBAMatrix((const uint16_t*)image->yuvPlanes[AVIF_CHAN_Y],
+                                 image->yuvRowBytes[AVIF_CHAN_Y] / 2,
+                                 (const uint16_t*)image->yuvPlanes[AVIF_CHAN_U],
+                                 image->yuvRowBytes[AVIF_CHAN_U] / 2,
+                                 (const uint16_t*)image->yuvPlanes[AVIF_CHAN_V],
+                                 image->yuvRowBytes[AVIF_CHAN_V] / 2,
+                                 rgb->pixels,
+                                 rgb->rowBytes,
+                                 matrixYUV,
+                                 image->width,
+                                 image->height) != 0) {
+                return AVIF_RESULT_REFORMAT_FAILED;
+            }
+            return AVIF_RESULT_OK;
+#endif
+        }
+        if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV422) {
+            // This doesn't currently exist in libyuv
+#if 0
+            if (I210ToRGBAMatrix((const uint16_t*)image->yuvPlanes[AVIF_CHAN_Y],
+                                 image->yuvRowBytes[AVIF_CHAN_Y] / 2,
+                                 (const uint16_t*)image->yuvPlanes[AVIF_CHAN_U],
+                                 image->yuvRowBytes[AVIF_CHAN_U] / 2,
+                                 (const uint16_t*)image->yuvPlanes[AVIF_CHAN_V],
+                                 image->yuvRowBytes[AVIF_CHAN_V] / 2,
+                                 rgb->pixels,
+                                 rgb->rowBytes,
+                                 matrixYUV,
+                                 image->width,
+                                 image->height) != 0) {
+                return AVIF_RESULT_REFORMAT_FAILED;
+            }
+            return AVIF_RESULT_OK;
+#endif
+        }
+        if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV420) {
+            // This doesn't currently exist in libyuv
+#if 0
+            if (I010ToRGBAMatrix((const uint16_t*)image->yuvPlanes[AVIF_CHAN_Y],
+                                 image->yuvRowBytes[AVIF_CHAN_Y] / 2,
+                                 (const uint16_t*)image->yuvPlanes[AVIF_CHAN_U],
+                                 image->yuvRowBytes[AVIF_CHAN_U] / 2,
+                                 (const uint16_t*)image->yuvPlanes[AVIF_CHAN_V],
+                                 image->yuvRowBytes[AVIF_CHAN_V] / 2,
+                                 rgb->pixels,
+                                 rgb->rowBytes,
+                                 matrixYUV,
+                                 image->width,
+                                 image->height) != 0) {
+                return AVIF_RESULT_REFORMAT_FAILED;
+            }
+            return AVIF_RESULT_OK;
+#endif
+        }
+        if (image->yuvFormat == AVIF_PIXEL_FORMAT_YUV400) {
+            // This doesn't currently exist in libyuv
```

Comment:
Note that unlike the above cases, I did not add an `#if 0` block here with some code, because I don't know what 10 bit 400 YUV (grey) will be called in libyuv.